### PR TITLE
Completed the mailroom assignment

### PR DESCRIPTION
**Pull request by jchando1**
For letter format and report generation i leveraged the solution code shared.

I still get a space before the first name when i print the list of existing donors, not sure why though!

### CODE REVIEW

**Comment by jchando1** [comment]
Reminder - this is still pending to be merged and evaluated

File: students/JasneetChandok/Lesson03/mailroom.py [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+          It will be put to very good use.
+
+                         Sincerely,
+                            -The Team
+          '''.format(new_donor_name, new_donor_donation))
+
+
+# Function for updating record and thanking existing donor
+def existing_donor():
+    fmt_donor = "\n".join(["{}"] * len(donors)).format(*(donor[0] for donor in donors))
```
Comment (PythonCHB) [inline students/JasneetChandok/Lesson03/mailroom.py:37]: fancy!

File: students/JasneetChandok/Lesson03/mailroom.py [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+    print("List of Existing Donors:\n", fmt_donor, "\n")
+    donor_selection = input("Enter donor full name to store the new donation value>>>")
+    donor_donation = int(input("\nEnter the 'Amount' donated>>>"))
+    if donor_selection.strip().lower() == donors[0][0]:
```
Comment (PythonCHB) [inline students/JasneetChandok/Lesson03/mailroom.py:41]: this is just checking if it's the first one....

File: students/JasneetChandok/Lesson03/mailroom.py [hunk 3]
```diff
@@ -0,0 +1,126 @@
...
+        print("Select one of the two category!!")
+
+
+# Function for sorting report by amount
+def sort_key(item):
+    return item[1]
+
+
+# Function for generating reporting
+def generate_report():
```
Comment (PythonCHB) [inline students/JasneetChandok/Lesson03/mailroom.py:80]: this all looks good.

File: students/JasneetChandok/Lesson03/mailroom.py [hunk 4]
```diff
@@ -0,0 +1,126 @@
...
+        elif answer == "3":
+            quit_program()
+        else:
+            print("Please select either 1,2 or 3")
+
+
+if __name__ == "__main__":
+    print("*****Welcome to the Mailroom Program!!*****\n")
+
+main_menu()
```
Comment (PythonCHB) [inline students/JasneetChandok/Lesson03/mailroom.py:126]: this should be indented to be in the `if __name__` block.

**Review by PythonCHB — APPROVED** [review]
do check your mailroom for using an existing donor other than the first one.

but otherwise, all looks good.

**Pull request merged**